app/grants/management/commands/sent_cart_reminder.py

```diff
@@ -51,7 +51,7 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
 
-        _, round_start_date, round_end_date, _ = get_clr_rounds_metadata()
+        _, round_start_date, round_end_date, _, _, _, _, _ = get_clr_rounds_metadata()
```

Comment:
@chibie This is a quick fix, but this method is extremely fragile. It feels like it should be returning an object, not a tuple that we can access data on without guessing how many and where the `_` should go. I approve this so we can fix the deploy, but let's open a new tech debt ticket in Jira to fix this.

Reply:
@ksolo yea makes perfect sense

Reply:
Using a named tuple would likely provide enough of an abstraction to prevent these types of bugs - https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
ticket is GITC-477